### ISSUE TITLE
Usage for command line jar executable is confusing

### ISSUE THREAD

**Issue by abhi92** [jamesdbloom/mockserver#227]
Hi,

I think the usage notes for printed while running the "jetty jar with dependencies" is mixing up serverPort & proxyPort sections. Shouldn't it be reverse as in serverPort should be the port for the MockServer and proxyPort should be the port for the proxy?

<pre>
   java -jar <path to mockserver-jetty-jar-with-dependencies.jar> [-serverPort <port>] [-proxyPort <port>] [-proxyRemotePort <port>] [-proxyRemoteHost <hostname>]

     valid options are:
<b>
        -serverPort <port>           Specifies the HTTP, HTTPS, SOCKS and HTTP
                                     CONNECT port for proxy. Port unification
                                     supports for all protocols on the same port.

        -proxyPort <port>            Specifies the HTTP and HTTPS port for the
                                     MockServer. Port unification is used to
                                     support HTTP and HTTPS on the same port.
</b>

        -proxyRemotePort <port>      Specifies the port to forward all proxy
                                     requests to (i.e. all requests received on
                                     portPort). This setting is used to enable
                                     the port forwarding mode therefore this
                                     option disables the HTTP, HTTPS, SOCKS and
                                     HTTP CONNECT support.

        -proxyRemoteHost <hostname>  Specified the host to forward all proxy
                                     requests to (i.e. all requests received on
                                     portPort). This setting is ignored unless
                                     proxyRemotePort has been specified. If no
                                     value is provided for proxyRemoteHost when
                                     proxyRemotePort has been specified,
                                     proxyRemoteHost will default to "localhost".

   i.e. java -jar ./mockserver-jetty-jar-with-dependencies.jar -serverPort 1080 -proxyPort 1090 -proxyRemotePort 80 -proxyRemoteHost www.mock-server.com
</pre>

**Comment by jamesdbloom** [comment]
That is correct, I will fix this once I finished working on another issue. 

Also I notice you are using a very out of date version of MockServer.  I would suggest you now use mockserver-netty now as the Jetty version is no longer supported.

**Comment by jamesdbloom** [comment]
fixed so now closing this change will appear once the next release is done
